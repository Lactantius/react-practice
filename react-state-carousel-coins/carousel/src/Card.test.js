import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";
import image1 from "./image1.jpg";

it("renders", () => {
  render(<Card src={image1} caption="A photo"/>);
});

it("matches snapshot", () => {
  const { asFragment } = render(<Card src={image1} caption="A photo"/>);
  expect(asFragment()).toMatchSnapshot();
});