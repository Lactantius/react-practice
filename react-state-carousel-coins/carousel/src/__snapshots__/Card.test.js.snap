// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      A photo
    </h4>
    <img
      alt="A photo"
      class="Card-image"
      src="image1.jpg"
    />
    <small
      class="Card-small"
    >
      Image  of .
    </small>
  </div>
</DocumentFragment>
`;
